nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/src/main/java/org/apache/nifi/jasn1/JASN1Reader.java

```diff
@@ -193,10 +219,23 @@ protected Collection<ValidationResult> customValidate(ValidationContext validati
     @OnEnabled
     public void onEnabled(final ConfigurationContext context) throws InitializationException {
         if (context.getProperty(ASN_FILES) != null && context.getProperty(ASN_FILES).isSet()) {
-            String[] asnFilesPaths = Arrays.stream(context.getProperty(ASN_FILES).evaluateAttributeExpressions().getValue().split(","))
-                .map(String::trim)
-                .toArray(String[]::new);
+            String asnFilesString = context.getProperty(ASN_FILES).evaluateAttributeExpressions().getValue();
```

Comment:
There are still a few lines that are missing finals.

Reply:
In this particular case, the variable can be re-assigned on line 229, so this is acceptable.